.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I'm not a huge fan of putting the `settings.json` in the repository as I often override it myself in every project with specific settings. Are these configurations really useful?

Reply:
I've removed the settings.json and added the extensions.json ;)